### ISSUE TITLE
Fix CVE-2021-34552 vulnerability

### ISSUE THREAD

**Issue by jmcruz1983** [python-pillow/Pillow#5649]
Using pillow library with python docker image.
Then following vulnerability is shown by docker scan tool [trivy](https://github.com/aquasecurity/trivy).

```
trivy image [name]

Pipfile.lock
=====================
Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 1)

+---------+------------------+----------+-------------------+---------------+---------------------------------------+
| LIBRARY | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |                 TITLE                 |
+---------+------------------+----------+-------------------+---------------+---------------------------------------+
| pillow  | CVE-2021-34552   | CRITICAL | 8.2.0             |               | python-pillow: buffer overflow        |
|         |                  |          |                   |               | in Convert.c because it               |
|         |                  |          |                   |               | allow an attacker to pass...          |
|         |                  |          |                   |               | -->avd.aquasec.com/nvd/cve-2021-34552 |
+---------+------------------+----------+-------------------+---------------+---------------------------------------+
``` 

Such vulnerability avoids image to be pulled from the docker registry.
And there is no a fix for it.

**Comment by radarhere** [comment]
This CVE was addressed in Pillow 8.3.0. See the release notes - https://pillow.readthedocs.io/en/stable/releasenotes/8.3.0.html

Is there anything further you would like done here?

**Comment by jmcruz1983** [comment]
Thanks!
all goo from my side,
I'll use the fixed version.
